### DISCUSSION TITLE
Documentation Website Update

### DISCUSSION THREAD

**Discussion by aburt2** [IDMIL/T-Stick#40] [Ideas]
The T-stick document has needed an update for quite some time. I've started working on the documentation website a couple of months ago but definitely can use some help with writing content. 
Link to docs repo: https://github.com/idmil/tstick-docs
Docs website: https://idmil.github.io/tstick-docs/

Specifically guides for artist/musicians rather than other engineers/designers. @kaseypocius @takujin @edumeneses, I'd definitely appreciate your advice and help with writing these pages
- [ ] Getting Started Guide
- [ ] T-Stick Gestures Guide
- [x] Link to some nice performances

There is also the todo list from the docs GitHub page:
- [x] Finish Detailed Technical Information for T-Stick 4GW/5GW
- [x] Finish writing justifications for the technical equirements (and move them under t-stick 5gw header)
- [x] Write Contributer's Guide
- [ ] Create page templates
- [x] Check all links are functioning
- [x] Check all images display properly
- [ ] Move information from [T-Stick Archive Github repository](https://github.com/IDMIL/T-Stick-Archive/tree/510e0442aa7dcef755de2b2bd9c8bcb03a8b61f5) for the older T-Stick Designs

I'll leave this discussion open for any other ideas on what kinds of documentation we are missing.

**Comment by edumeneses** [comment]
Thanks for the initiative @aburt2 !

I suggest the gestural part should link to the future puara-gestures documentation. We will officially start a new project having Puara as a central part in Feb 2024, and I expect a lot of development to start soon. At this point, we can port or link Alex Nieva's thesis, which AFAIK contains info on the current gestures. 

I'll be back next week and can work on the gestural part

**Reply by aburt2** [reply]
I'll have a look at Alex's thesis and start extracting the gesture descriptions from their, probably in early to mid February
